django/core/mail/backends/filebased.py

```diff
@@ -44,10 +44,13 @@ def __init__(self, *args, file_path=None, **kwargs):
         super().__init__(*args, **kwargs)
 
     def write_message(self, message):
-        self.stream.write(message.message().as_bytes() + b'\n')
+        self.stream.write(b'%s\n' % self.format_message(message))
         self.stream.write(b'-' * 79)
         self.stream.write(b'\n')
 
+    def format_message(self, message):
```

Comment:
I think `format_message()` should always return a `str`. `write_message()` can `encode()` that if need be. 

Q: Why exactly are we writing bytes here? (NTS: see what fails if you can change this. 🙂)